### DISCUSSION TITLE
Mova E30 Ultra

### DISCUSSION THREAD

**Discussion by xeboman** [Tasshack/dreame-vacuum#800] [Q&A]
s there a possibility that Mova E30 Ultra will be integrated in the near future?

**Comment by OMEGAJZ** [comment]
> s there a possibility that Mova E30 Ultra will be integrated in the near future? 

+1

**Comment by Bjarne-Kinkel** [comment]
Please!!! Even if it's just a simple "start cleaning" without any fancy stuff, at first.

**Comment by Tasshack** [comment]
Supported on Beta.

**Reply by xeboman** [reply]
Thank you, I have just seen it. Unfortunately, the following error occurs when I try to log in to the Dreamehome Cloud: “Login not possible, please check the login data.”   However, the user and password are correct.

**Reply by Tasshack** [reply]
You need to use the Dreamehome app not Mova

**Reply by xeboman** [reply]
Thank you...app is installed. But now I get the following error message “No supported devices were found in this account for the selected country.” I selected 'EU' because I live in Switzerland. Does it have to do with the fact that Switzerland is not in the EU? 
Even if I select the Germany region in the Dreame APP, which is in the EU, it does not work.

**Comment by Tasshack** [comment]
https://github.com/Tasshack/dreame-vacuum/issues/892
